pkg/server/proto/server.proto

```diff
@@ -592,8 +592,10 @@ message UI {
   }
 
   message GetPipelineRunResponse {
+    // deprecated in favor of pipeline_run_bundle.
     PipelineRun pipeline_run = 1;
```

Comment:
**Why no `deprecated_` prefix?**

Though changing the name of a field is backwards-compatible in end-to-end protobuf, it’s not backwards-compatible with Swagger and our generated client library.

Reply:
> What, really? That's surprising. That kind of breaks the one reason why protobufs are cool and useful for crafting APIs. It should be safe to rename fields in protobufs as long as the number stays the same. If we lose that, then that's a shame 😞

Absolutely. All is good when you stay within gRPC. But unfortunately because we go via Swagger and the OpenAPI generator, we lose that delightful property of protobufs.

> Also if we're deprecating this, then we're gonna need to update everywhere in the system that uses this field. I know much of the CLI is relying on this field existing.

Oh, is the CLI using `UI_GetPipelineRun`?

Reply:
Ohhh shoot, I think the git diff wasn't showing me the full picture, but isn't this modifying the GetPipelineRunResponse?

Reply:
Ahhh, looks like this is within UI? So maybe not, all good I bet!

Reply:
Yeah, should just be `UI.GetPipelineRunResponse`.